ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -62,6 +62,21 @@ tests:
           cpu: "2"
           memory: 4Gi
     workflow: ipi-aws
+- as: e2e-console
+  run_if_changed: frontend/
+  steps:
+    cluster_profile: gcp-openshift-gce-devel-ci-2
+    test:
+    - as: console_test
+      cli: latest
+      commands:
+        frontend/test-frontend.sh
```

Comment:
It's better to name it with `console-test-frontend.sh`, more: https://docs.ci.openshift.org/docs/architecture/step-registry/#registry-layout-and-naming-convention

Reply:
the test script has been merged in our openshift-tests-private repo, how about keep it as it is? @jianzhangbjz

Reply:
@yapei it is not same name with https://github.com/openshift/openshift-tests-private/pull/2246.
and only https://github.com/openshift/openshift-tests-private/pull/2246 is merged, we could approve it.